backend/src/main/java/gov/cdc/usds/simplereport/api/apiuser/ApiUserMutationResolver.java

```diff
@@ -57,9 +62,10 @@ public User updateUser(
             _us.assertEmailAvailable(newEmail);
         }
         ApiUser apiUser = _us.updateUser(newEmail, oldEmail, firstName, middleName, lastName, suffix);
-        Organization org = _os.getOrganizationForUser(apiUser);
+        Optional<Organization> org = _os.getOrganizationForUser(apiUser);
         Boolean isAdmin = _us.isAdminUser(apiUser);
-        return new User(apiUser, org, isAdmin);
+		List<UserPermission> permissions = _us.getPermissions(apiUser);
+		return new User(apiUser, org, isAdmin, permissions);
```

Comment:
In order to do this _right_ we will need to get the group memberships for this org for this user, and translate them into permissions—and I was thinking we could just remove that from the contract for this mutation, but in the medium term we will want to be able to change the user's role through this mutation and we will want to be able to list users with their roles, so that won't do. On the other hand in the _short_ term we're not using this mutation anyway, so cowboying it seems most appropriate.

Reply:
ah, i wasn't clear on the fact that group memberships were the determiner of permissions. got it.